client/client_test.go

```diff
@@ -1959,6 +1960,26 @@ func testBasicLocalCacheImportExport(t *testing.T, sb integration.Sandbox) {
 	testBasicCacheImportExport(t, sb, []CacheOptionsEntry{im}, []CacheOptionsEntry{ex})
 }
 
+func testCustomTagLocalCacheImportExport(t *testing.T, sb integration.Sandbox) {
+	dir, err := ioutil.TempDir("", "buildkit")
+	require.NoError(t, err)
+	defer os.RemoveAll(dir)
+	im := CacheOptionsEntry{
+		Type: "local",
+		Attrs: map[string]string{
+			"src": dir,
+			"tag": "customtag",
```

Comment:
undefined tag

Reply:
I deleted test code because I didn't know how to write a test using a custom tag.